<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ImageView" name="graphicsView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="pageFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="pageLabel">
         <property name="text">
          <string>any folder or archive is not loaded.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PageSlider" name="pageSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuHistory">
     <property name="title">
      <string>&amp;History</string>
     </property>
    </widget>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="menuHistory"/>
    <addaction name="actionClearHistory"/>
    <addaction name="actionAutoLoaded"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionScaleUp"/>
    <addaction name="actionScaleDown"/>
    <addaction name="actionFitting"/>
    <addaction name="separator"/>
    <addaction name="actionDualView"/>
    <addaction name="actionRightSideBook"/>
    <addaction name="actionWideImageAsOneView"/>
    <addaction name="actionFirstImageAsOneView"/>
    <addaction name="separator"/>
    <addaction name="actionShowMenuBar"/>
    <addaction name="separator"/>
    <addaction name="actionShowToolBar"/>
    <addaction name="actionShowPageBar"/>
    <addaction name="actionShowStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionStayOnTop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOpenKeyConfig"/>
    <addaction name="separator"/>
    <addaction name="actionCheckVersion"/>
    <addaction name="actionAppVersion"/>
   </widget>
   <widget class="QMenu" name="menu_Navigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionNextPage"/>
    <addaction name="actionPrevPage"/>
    <addaction name="actionFirstPage"/>
    <addaction name="actionLastPage"/>
    <addaction name="actionNextOnePage"/>
    <addaction name="actionPrevOnePage"/>
    <addaction name="separator"/>
    <addaction name="actionNextVolume"/>
    <addaction name="actionPrevVolume"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Navigation"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevPage"/>
   <addaction name="actionNextPage"/>
   <addaction name="actionFirstPage"/>
   <addaction name="actionLastPage"/>
   <addaction name="separator"/>
   <addaction name="actionPrevVolume"/>
   <addaction name="actionNextVolume"/>
   <addaction name="separator"/>
   <addaction name="actionScaleUp"/>
   <addaction name="actionScaleDown"/>
   <addaction name="separator"/>
   <addaction name="actionFitting"/>
   <addaction name="actionDualView"/>
   <addaction name="separator"/>
   <addaction name="actionFullscreen"/>
   <addaction name="separator"/>
   <addaction name="actionStayOnTop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/page_next_icon2</normaloff>:/icons/page_next_icon2</iconset>
   </property>
   <property name="text">
    <string>&amp;NextPage</string>
   </property>
   <property name="toolTip">
    <string>NextPage</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionPrevPage">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/page_previous_icon2</normaloff>:/icons/page_previous_icon2</iconset>
   </property>
   <property name="text">
    <string>&amp;PrevPage</string>
   </property>
   <property name="toolTip">
    <string>PrevPage</string>
   </property>
  </action>
  <action name="actionScaleUp">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/power_plus_icon</normaloff>:/icons/power_plus_icon</iconset>
   </property>
   <property name="text">
    <string>Scale&amp;Up</string>
   </property>
  </action>
  <action name="actionScaleDown">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/power_minus_icon</normaloff>:/icons/power_minus_icon</iconset>
   </property>
   <property name="text">
    <string>Scale&amp;Out</string>
   </property>
   <property name="toolTip">
    <string>ScaleOut</string>
   </property>
  </action>
  <action name="actionFitting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/maximize_icon</normaloff>:/icons/maximize_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Fitting</string>
   </property>
   <property name="toolTip">
    <string>Fitting</string>
   </property>
  </action>
  <action name="actionDualView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/dualview</normaloff>:/icons/dualview</iconset>
   </property>
   <property name="text">
    <string>&amp;Spread Viewing</string>
   </property>
   <property name="iconText">
    <string>Spread Viewing</string>
   </property>
   <property name="toolTip">
    <string>Spread Viewing</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/fullscreen2</normaloff>:/icons/fullscreen2</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionAppVersion">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
   </property>
   <property name="text">
    <string>&amp;About QuickViewer</string>
   </property>
  </action>
  <action name="actionAutoLoaded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Auto load last used</string>
   </property>
  </action>
  <action name="actionRightSideBook">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bound on the &amp;right side book</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>&amp;Open Image Folder</string>
   </property>
  </action>
  <action name="actionWideImageAsOneView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Always open wide images as one view</string>
   </property>
  </action>
  <action name="actionLastPage">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/page_last_icon</normaloff>:/icons/page_last_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;LastPage</string>
   </property>
  </action>
  <action name="actionFirstPage">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/page_first_icon</normaloff>:/icons/page_first_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;FirstPage</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ShowStatus&amp;Bar</string>
   </property>
  </action>
  <action name="actionShowPageBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show&amp;PageBar</string>
   </property>
  </action>
  <action name="actionOpenFiler">
   <property name="text">
    <string>Open &amp;Explorer</string>
   </property>
  </action>
  <action name="actionOpenExif">
   <property name="text">
    <string>Exif &amp;Information</string>
   </property>
  </action>
  <action name="actionOpenKeyConfig">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/preference_icon</normaloff>:/icons/preference_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Key Config</string>
   </property>
  </action>
  <action name="actionCheckVersion">
   <property name="text">
    <string>Check &amp;Version by your browser</string>
   </property>
  </action>
  <action name="actionNextVolume">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/volume_next_icon</normaloff>:/icons/volume_next_icon</iconset>
   </property>
   <property name="text">
    <string>N&amp;extVolume</string>
   </property>
  </action>
  <action name="actionPrevVolume">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/volume_prev_icon</normaloff>:/icons/volume_prev_icon</iconset>
   </property>
   <property name="text">
    <string>P&amp;revVolume</string>
   </property>
  </action>
  <action name="actionNextOnePage">
   <property name="text">
    <string>Next only &amp;one page</string>
   </property>
  </action>
  <action name="actionPrevOnePage">
   <property name="text">
    <string>Prev only o&amp;ne page</string>
   </property>
  </action>
  <action name="actionExitApplicationOrFullscreen">
   <property name="text">
    <string>ExitApplicationOrFullscreen</string>
   </property>
  </action>
  <action name="actionShowToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show&amp;ToolBar</string>
   </property>
  </action>
  <action name="actionFirstImageAsOneView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always open &amp;first images as one view</string>
   </property>
  </action>
  <action name="actionShowMenuBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show&amp;MenuBar</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="text">
    <string>&amp;Clear History</string>
   </property>
  </action>
  <action name="actionStayOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/stayontop_icon</normaloff>:/icons/stayontop_icon</iconset>
   </property>
   <property name="text">
    <string>Stay On &amp;Top</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>imageview.h</header>
   <slots>
    <slot>on_image_changing(QImage)</slot>
    <slot>on_fitting_triggered(bool)</slot>
    <slot>on_dualView_triggered(bool)</slot>
    <slot>on_scaleUp_triggered()</slot>
    <slot>on_scaleDown_triggered()</slot>
    <slot>on_rightSideBook_triggered(bool)</slot>
    <slot>on_wideImageAsOneView_triggered(bool)</slot>
    <slot>on_openFiler_triggered()</slot>
    <slot>on_openExifDialog_triggered()</slot>
    <slot>on_nextPage_triggered()</slot>
    <slot>on_prevPage_triggered()</slot>
    <slot>on_firstPage_triggered()</slot>
    <slot>on_lastPage_triggered()</slot>
    <slot>on_nextOnlyOnePage_triggered()</slot>
    <slot>on_prevOnlyOnePage_triggered()</slot>
    <slot>on_firstImageAsOneView_triggered(bool)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>PageSlider</class>
   <extends>QSlider</extends>
   <header>pageslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="toolbar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNextPage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_nextPage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevPage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_prevPage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScaleUp</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_scaleUp_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScaleDown</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_scaleDown_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFitting</sender>
   <signal>triggered(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>on_fitting_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDualView</sender>
   <signal>triggered(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>on_dualView_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_fullscreen_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAppVersion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_appVersion_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAutoLoaded</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_autoloaded_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRightSideBook</sender>
   <signal>triggered(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>on_rightSideBook_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenFolder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_openfolder_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWideImageAsOneView</sender>
   <signal>triggered(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>on_wideImageAsOneView_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFirstPage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_firstPage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLastPage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_lastPage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowPageBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_showSliderBar_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowStatusBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_showStatusBar_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenExif</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_openExifDialog_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenFiler</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_openFiler_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenKeyConfig</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_openKeyConfig_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCheckVersion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_checkVersion_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextVolume</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_nextVolume_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevVolume</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_prevVolume_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextOnePage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_nextOnlyOnePage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevOnePage</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>on_prevOnlyOnePage_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExitApplicationOrFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_exitApplicationOrFullscreen_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_exit_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowToolBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_showToolBar_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFirstImageAsOneView</sender>
   <signal>triggered(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>on_firstImageAsOneView_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowMenuBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_showMenuBar_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearHistory</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_clearHistory_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStayOnTop</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_stayOnTop_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_file_changed(QString)</slot>
  <slot>on_nextPage_triggered()</slot>
  <slot>on_prevPage_triggered()</slot>
  <slot>on_scaleUp_triggered()</slot>
  <slot>on_scaleDown_triggered()</slot>
  <slot>on_fullscreen_triggered()</slot>
  <slot>on_appVersion_triggered()</slot>
  <slot>on_autoloaded_triggered(bool)</slot>
  <slot>on_openfolder_triggered()</slot>
  <slot>on_firstPage_triggered()</slot>
  <slot>on_lastPage_triggered()</slot>
  <slot>on_showSliderBar_triggered(bool)</slot>
  <slot>on_showStatusBar_triggered(bool)</slot>
  <slot>on_openKeyConfig_triggered()</slot>
  <slot>on_checkVersion_triggered()</slot>
  <slot>on_nextVolume_triggered()</slot>
  <slot>on_prevVolume_triggered()</slot>
  <slot>on_exitApplicationOrFullscreen_triggered()</slot>
  <slot>on_exit_triggered()</slot>
  <slot>on_showToolBar_triggered(bool)</slot>
  <slot>on_showMenuBar_triggered(bool)</slot>
  <slot>on_clearHistory_triggered()</slot>
  <slot>on_stayOnTop_triggered(bool)</slot>
 </slots>
</ui>
